lib/gcs_signer.ex

```diff
@@ -5,12 +5,14 @@ defmodule GcsSigner do
 
   @base_url "https://storage.googleapis.com"
 
+  @otp_greater_21? :erlang.system_info(:otp_release) >= '21'
```

Comment:
Is it possible to change single quote to double quote, since this project uses double quote as a convention?

Reply:
`:erlang.system_info(:otp_release)` returns an erlang charlist, which in elixir is denoted by single quotes. If I were to compare with a Binary (double quotes), the result would always be false:
```
iex> '21' >= "21"
false
```

Reply:
Ah I see! Thanks for the insight 😄